pkg/util/conntrack/conntrack.go

```diff
@@ -82,11 +86,21 @@ func Exists(execer exec.Interface) bool {
 // https://github.com/docker/docker/issues/8795
 // https://github.com/kubernetes/kubernetes/issues/31983
 func ClearEntriesForPort(execer exec.Interface, port int, isIPv6 bool, protocol v1.Protocol) error {
+	return clearPortEntriesInternal(execer, "", port, isIPv6, protocol)
+}
+
+// ClearEntriesForHostPort is similar to the ClearEntriesForPort function, but uses conntrack binary installed at a
+// custom location.
```

Comment:
Why should conntrack entry clearing work for this use case, but still fail in all the other places it's called?

Reply:
Are you asking why we need a custom conntrack binary here, but in other places it's called, it is picked up from PATH?
That's because the other calls are from kubeproxy running in a container that has conntrack installed. hostport implementation runs on the node and the image does not have conntrack. Sorry if i misunderstood the question.